airbyte-cdk/python/pyproject.toml

```diff
@@ -66,7 +66,9 @@ pytz = "2024.1"
 [tool.poetry.group.dev.dependencies]
 freezegun = "*"
 mypy = "*"
-pandas = "2.0.3"
+asyncio = "*"
+pandas = "2.2.0"
+python-calamine = "^0.2.3"
```

Comment:
I suspect this should be in the `file-based` extra

Reply:
done